css/index.css

```diff
@@ -112,6 +203,46 @@ p {
     justify-content: center;
 }
 
```

Comment:
If you can I would avoid using id unless you absolutely have to. You could have done something similar by using .boxes .box .box1 and then .boxes .box .box2 and so on